suse-cloud-image-name-parser.spec

```diff
@@ -0,0 +1,57 @@
+#
+# spec file for package suse-cloud-image-name-parser
+#
+# Copyright (c) 2025 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+%define upstream_name suse-cloud-image-name-parser
+%{?sle15_python_module_pythons}
```

Comment:
Should we build this as module, i.e. use `%python_subpackages` and subsequentlt end up with python$VERSION-$NAME binary packages or does it make more sense for use to stick with 1 interpreter?

Building with `%python_subpackages` may reduce the coordination effort when other bits that get updated move to a newer Python interpreter, but it also means that when we incude this in image builds we create churn in the image descriptions as we haveto chase the $VERSION binary name.

Reply:
I don't think this has any usefulness inside of an image. The parser requires the cloud image name which isn't really accessible from inside an image.